components/MovieTile/MovieTile.js

```diff
@@ -0,0 +1,92 @@
+"use client"
+import React, {useContext, useEffect, useState} from "react";
+import { useRouter } from 'next/navigation'
+import styles from './MovieTile.module.scss';
+import Image from "next/image";
+import Link from "next/link";
+import {auto} from "../../storybook-static/sb-manager/chunk-CXYKRFSY";
```

Comment:
I see that this is an unused import. However, it breaks the app running/building if you don't have 'storybool-static' folder in the root. I didn't have it, so I faced the error (fixed locally).

Reply:
Thanks, I did not notice this. Removed.